web-app-router/scenes/Portal/Teams/TeamId/Team/page/Members/List/PermissionsDialog/index.tsx

```diff
@@ -0,0 +1,176 @@
+import { Button } from "@/components/Button";
+import { Dialog } from "@/components/Dialog";
+import { DialogOverlay } from "@/components/DialogOverlay";
+import { DialogPanel } from "@/components/DialogPanel";
+import { ArrowRightIcon } from "@/components/Icons/ArrowRightIcon";
+import { CheckmarkCircleIcon } from "@/components/Icons/CheckmarkCircleIcon";
+import { TYPOGRAPHY, Typography } from "@/components/Typography";
+import { Role_Enum } from "@/graphql/graphql";
+import clsx from "clsx";
+import { atom, useAtom } from "jotai";
+
+export const permissionsDialogAtom = atom(false);
+
+type PermissionsConfig = Record<
+  string,
+  Record<keyof typeof Role_Enum, boolean>
+>;
+
+const config: PermissionsConfig = {
+  "Create & Edit incognito actions": {
+    Owner: true,
+    Admin: true,
+    Member: false,
```

Comment:
Was incorrect in figma 
```suggestion
    Member: true,
```